app-config-webpack/src/loader.ts

```diff
@@ -19,6 +20,29 @@ const loader = function AppConfigLoader(this: Loader) {
   const loadingOptions = options.loadingOptions ?? options.loading ?? {};
   const schemaLoadingOptions = options.schemaLoadingOptions ?? options.schemaLoading;
   const injectValidationFunction = options.injectValidationFunction ?? true;
+  const doNotLoadConfig = options.doNotLoadConfig ?? false;
```

Comment:
Thoughts on something like `skipBuildTimeConfig`? I can see people getting confused thinking this will prevent config from loading altogether somehow.

Reply:
That's a good point - `doNotBundleConfig`, `noBuildTimeConfig`, `noBundledConfig`?

Reply:
I like `noBundledConfig` and `doNotBundleConfig`. Though can't say I've seen the `doNot...` convention before.